guides/v3.7.0/reference/accessibility.md

```diff
@@ -0,0 +1,96 @@
+The purpose of this guide is to give every Ember developer a starting point for making your Ember application minimally accessible. As accessibility can be a complex issue, it should be noted that this guide only covers a some of the basics, and should not be considered a comprehensive guide. 
+
+## Screen Reader and Browser Combinations
+
+There are assistive technologies (known as screen readers) available for all common desktop platforms and mobile devices. 
+
+- VoiceOver, integrated in Apple products
+- Narrator, integrated in Windows products
+- Orca, available for integration in Ubuntu, otherwise available as a download
+- JAWS, proprietary software by Freedom Scientific, available for Windows
+- NVDA, open source software, available for Windows
+- Talkback, integrated in Android products
+
+While developing and testing for conformance, keep in mind that there are well-known screen reader and browser combinations that were developed in a way that work well together; using combinations different than these may produce false-positive results. It should be noted that these may change over time, so periodic review of this list is recommended. 
+
+- Firefox & NVDA (Windows)
+- IE & JAWS (Windows)
+- Safari & VoiceOver (MacOS)
+
+The absolute best method for learning how a screen reader works is using one yourself! It might feel a little awkward at first, but understanding how to use a screen reader (and other assistive technology) will help you become a more skilled developer. 
+
+## Ember Application Configuration
+
+### Ember Application vs role="application"
+
+The first thing that should be discussed in this section is this: "application" in Ember development and "application" in landmark roles have two _very_ different meanings. 
+
+The TL;DR? Don't use `role="application"` until you have done your research and know exactly how it is to be used correctly (if at all). There are very few use cases where the role of application is appropriate. 
+
+Read more about it: [https://a11yproject.com/posts/how-to-use-application-role/](https://a11yproject.com/posts/how-to-use-application-role/)
+
+
+### Optional Feature: Application Template Wrapper
+
+If you are using the [application template wrapper](#toc_optional-features) enabled (default state), then you will need to add certain aria roles to your [landmark regions](https://www.w3.org/WAI/PF/aria/roles#landmark_roles), even if you are using native HTML elements, because those regions are not the direct child descendant of the body element (they are the children of the div that wraps the Ember app).
```

Comment:
It's not clear from this description whether having the application template wrapper is a good thing or not. I think it should recommend for or against it as a way to improve accessibility, right away. In general, I would say always show first the option that is the "better" choice.